server/src/model/seed.js

```diff
@@ -4,20 +4,22 @@ import { connectionString } from '../config/config';
 
 const hashedPassword = bcryptjs.hashSync('test', 8);
 
-const seedUserData = { text: 'INSERT INTO users (firstname, lastname, email, password, role) VALUES($1, $2,$3,$4,$5)',
-  values: ['Sinmi', 'John', 'sinmiloluwasunday@yahoo.com', hashedPassword, 'admin'] };
+const seedUserData = {
+  text: 'INSERT INTO users (firstname, lastname, email, password, role) VALUES($1, $2,$3,$4,$5)',
+  values: ['Sinmi', 'John', 'sinmiloluwasunday@yahoo.com', hashedPassword, 'admin']
+};
 const seedRequestData1 = `INSERT INTO requests (title, description, category, image, status, dated, userId) VALUES
-('Faulty Fan', 'we have a fault', 'electrical', 'IMAGE', 'approved', '2018-12-13', 1);`;
+('Damaged Chair', 'we have a fault', 'electrical', 'chair.jpg', 'approved', '2018-12-13', 1);`;
```

Comment:
use cloudinary links to render these images.

Reply:
alright I will, thanks